netci.groovy

```diff
@@ -6,6 +6,8 @@ def project = GithubProject
 
 // Globals
 
+// List of all CI OS
+def allOSs = ['Ubuntu', 'Debian8.2', 'OSX', 'FreeBSD', 'CentOS7.1', 'OpenSUSE13.2', 'Windows 10', 'Windows 7', 'Windows_NT']
```

Comment:
Where is this used? Perhaps I missed it but I don't see it used in the script anywhere.